src/components/look-controls.js

```diff
@@ -28,7 +28,11 @@ module.exports.Component = registerComponent('look-controls', {
 
   play: function () {
     this.previousPosition.set(0, 0, 0);
-    this.addEventListeners();
+
+    // defer adding listeners until everything has run.
```

Comment:
Same issue with the `Cannot read property 'render' of undefined` error in the console in the same issue.   

If I checked `scene.hasLoaded` it's ` false`.   Resolves to `true` in FF.   Just a difference in the order in which the methods are called In Chrome?